FreeRTOS-Plus/Source/Application-Protocols/readme.txt

```diff
@@ -0,0 +1,15 @@
+The 'core' branded application protocols are 'standalone' in that they do not
+have any dependencies outside of the C library.  They use a simple transport
+interface definition to ensure they are not dependent on the underlying TCP/IP
+stack.  This directory collects together the application protocols that all use
+the same transport interface definition (only coreMQTT at the time of writing,
+soon to also include coreHTTP).
+
+Directories:
+
++ coreMQTT contains the implementation of the coreMQTT library.  See:
+  https://www.FreeRTOS.org/coremqtt
```

Comment:
Doesn't this need an `/index.html`?

Reply:
As per other comments of the same, the web browser will add /index.html automatically, so its not needed.